instrumentation/opentelemetry-instrumentation-kafka-python/src/opentelemetry/instrumentation/kafka/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+_instruments = ("kafka-python >= 2.0",)
```

Comment:
Just curious, what is the difference between [kafka](https://pypi.org/project/kafka/) and [kafka-python](https://pypi.org/project/kafka-python/)? Will this only instrument `kafka-python`?

Reply:
they are the same project, but newer versions are released as kafka-python (latest 'kafka' release was in 2017)

Reply:
Curious should we rename all instances of `kafka` to `kafka-python` then? e.g. import path, tox environments, entrypoint, etc.

Reply:
I've tried to keep with the kafka library  conventions, where the package name is `kafka-python` but you actually use `kafka` in the import paths